### DISCUSSION TITLE
npm run dev Doesn't work !!!

### DISCUSSION THREAD

**Discussion by fouedd9** [vercel/next.js#14869] [Help]
i'm following the documentation of next.js, and when i arrived to the chapter **Pre-rendering and Data Fetching** the npm run dev doesn't work anymore after i install **npm install remark remark-html** 

I'm Stuck !!! 

Anyone can help me please to continue learning/following the next documentation ( next.js/blog post ) 

Thank you so much ✌🏻

![Capture d’écran 2020-07-05 à 19 05 39](https://user-images.githubusercontent.com/44516197/86537880-8c187e80-bef2-11ea-8c08-711c0926e4a4.png)

**Comment by stephanelpaul** [comment]
try running `npm i` then running `npm run dev`, just looks like your missing some node go higher in the error message you see the MODULE_NOT_FOUND it should indicate what it is a bit higher in the terminal.

**Reply by fouedd9** [reply]
Couldn't find a `pages` directory. Please create one under the project root ! 

thank you for helping me !!

**Comment by perryhq** [comment]
This is what appears when I run npm run dev, which is configured to do next dev:
![image](https://user-images.githubusercontent.com/62119804/146494128-7addf651-71ea-45d5-9dd3-896cc517c28c.png)
My packages are:
![image](https://user-images.githubusercontent.com/62119804/146494257-6cbff42a-28d2-483e-a6fa-429c2c6075c1.png)
